contracts/lending/Flashloan.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+import "./interfaces/IFlashloan.sol";
+
+contract Flashloan is IFlashloan, Ownable {
```

Comment:
Let's add the following functions:

withdrawETH(uint256 _amount) onlyOwner

withdrawERC20(address contractAddress, uint256_ amount) onlyOwner

withdrawERC721(address contractAddress, uint256_ amount) onlyOwner

Reply:
this too please